content/en/blog/_posts/2024-05-13-gateway-api-v1.1/index.md

```diff
@@ -0,0 +1,267 @@
+  Arko Dasgupta (Tetrate),
+  Flynn (Buoyant),
+  Candace Holman (Red Hat),
+  John Howard (Solo.io),
+  Mattia Lavacca (Kong),
+  Christine Kim (Isovalent),
+  Grant Spence (Red Hat),
+  Gina Yeh (Google),
+  and other reviewer and release note contributors.
+---
+
+![Gateway API Logo](gateway-api-logo.svg)
+
+In the wake of the GA release of Gateway API last October, the Kubernetes
+SIG Network buoyantly announces the v1.1 release of
+[Gateway API](https://gateway-api.sigs.k8s.io/). Gateway API features for
+**Gateway API for Service Mesh (GAMMA)**, **GRPCRoute**, **ParentReference Port**,
+and **Conformance Profiles and Reports** will graduate to Standard.
+In addition, **Gateway Client Certificate Verification** and **Session
+Persistence** are adopted as new, Experimental features. Clarifications to TLS
+terminology and gateway merging are added to strengthen the release.
+
+## What's new
+
+### Graduation to Standard
+This release includes the graduation to _Standard_ of four eagerly awaited features.
+This means they are no longer experimental concepts. Inclusion in the Standard
+release channel denotes a high
+level of confidence in the API surface and provides guarantees of backward
+compatibility.  Though these features are now considered stable, that does not
+mean that they are complete. These and other features will continue to evolve via
+the Experimental channel. For more information on how all of this works, refer
+to the [Gateway API Versioning Policy](https://gateway-api.sigs.k8s.io/concepts/versioning/).
+
+
+#### [GAMMA](https://gateway-api.sigs.k8s.io/mesh/)
+Service mesh support in Gateway API allows service mesh users to use the same
+API to manage ingress traffic and mesh traffic, reusing the same policy and
+routing interfaces. In Gateway API v1.1, routes (such as HTTPRoute) can now have
+a Service as a `parentRef`, to control how traffic to specific services behave.
+For more information, read the 
+[Gateway API service mesh documentation](https://gateway-api.sigs.k8s.io/mesh/)
+or see the
+[list of Gateway API implementations
+](https://gateway-api.sigs.k8s.io/implementations/#service-mesh-implementation-status).
+
+As an example, one could do a canary deployment of a workload deep in an
+application's call graph with an HTTPRoute as follows:
+
+```
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: color-canary
+  namespace: faces
+spec:
+  parentRefs:
+    - name: color
+      kind: Service
+      group: core
+      port: 80
+  rules:
+  - backendRefs:
+    - name: color
+      port: 80
+      weight: 50
+    - name: color2
+      port: 80
+      weight: 50
+```
+      
+This would split traffic sent to the `color` Service in the `faces` namespace
+50/50 between the original `color` Service and the `color2` Service, using a
+portable configuration that's easy to move from one mesh to another.
+
+Leading Contributors: Flynn, John Howard, Keith Mattix, Mike Morris
```

Comment:
Complete optional, but could be a nice touch to link to GitHub profiles

Reply:
```suggestion
Leading Contributors: [Flynn](https://github.com/kflynn), [John Howard](https://github.com/howardjohn/), [Keith Mattix](https://github.com/keithmattix/), [Mike Morris](https://github.com/mikemorris/)
```

Reply:
The GitHub profile are listed in the Release Notes already, so I'd iike to call this done with my last round of changes.

Reply:
I agree that this should not block the PR, but if someone has the time to add suggestions for all names throughout the PR, I think that would be a helpful addition, I'm not sure how many people will bother to click through to the release notes after reading this.

Reply:
I've decided to remove the Leading Contributors lines.  There are no other Gateway API blog articles where we list contributors, and I would prefer to highlight the team as a whole.  No hard feelings felt by anyone, I hope.

Reply:
Since I have to make other changes, I will add the github profile links to the authors list.  This seems to be the way others are handling the attributions.